pages/morse.js

```diff
@@ -71,7 +72,9 @@ export default function ButtonAppBar() {
                 <Typography variant="h5">tečka</Typography>
               </Button>
               <Button variant="outlined" onClick={handle_separator_click}>
-                <SendIcon />
+                <Typography fontSize="1000%" fontWeight="bold">
+                  /
+                </Typography>
```

Comment:
Separator is too big, label is outside of the button.
![image](https://user-images.githubusercontent.com/2271850/203840595-f72c7099-5fcf-4752-a97e-6c5abb0c935d.png)

See `.buttons > * > svg` CSS rule in index.modle.css. It can be similar here, e.g. `font-size: max(80px, 20vmin); line-height: 1;` And also `font-weight: bold;`.